### ISSUE TITLE
very slow to render

### ISSUE THREAD

**Issue by keturn** [standardebooks/lewis-carroll_through-the-looking-glass_john-tenniel#10]
It is very slow to open chapters and pages of this (kepub) on my Kobo Clara HD. Long enough that I thought it had completely frozen the device at first. Given that loading https://github.com/standardebooks/lewis-carroll_through-the-looking-glass_john-tenniel/blob/master/src/epub/images/illustration-2.svg severely lags my chromebook for at least a good five seconds or so, I think it's a pretty safe to blame the large SVGs.

How were they vectorized? Is there a way to slim them down?

**Comment by acabal** [comment]
Yes, the SVGs are quite large. Open to suggestions on ways to slim them down.

**Comment by keturn** [comment]
I've turned up a couple packages that might be relevant:
- https://github.com/luncheon/simplify-svg-path
- https://github.com/mourner/simplify-js

But I think that before we go trying to wrangle these generated svg files to something manageable, it could be worth re-examining how they were generated in the first place. Are there parameters on that end for adjusting the number or complexity of the paths it spits out?

**Comment by acabal** [comment]
Closed due to inactivity.
